src/utils/dates.js

```diff
@@ -10,3 +10,11 @@ const ONE_DAY_IN_MILLISECONDS = 86400000;
 export function getFutureDate(offset) {
 	return new Date(Date.now() + offset * ONE_DAY_IN_MILLISECONDS);
 }
+
+export function getDaysBetweenDates(startTime, currentTime) {
+	// param both in ms
+	const daysSinceLastPurchase = Math.round(
+		(currentTime - startTime) / ONE_DAY_IN_MILLISECONDS,
+	);
+	return daysSinceLastPurchase;
+}
```

Comment:
Yay for more utils!

Reply:
Nice! I love the extractions into the utils folder 🔥 !